### ISSUE TITLE
Passphrase recover.

### ISSUE THREAD

**Issue by woodsjesse607** [keybase/keybase-issues#3705]
How can

**Comment by Brianetta** [comment]
You need a device or a paper key. The only alternative is to reset your account, which will recover your username, but nothing else.

**Comment by woodsjesse607** [comment]
I have gotten back into my account but is there anyway to get the app to recognize my device after  reinstalling the app?

**Comment by Brianetta** [comment]
Which of the following do you have?

- Password
- Paper Key
- An alternative, working device

**Comment by woodsjesse607** [comment]
Password and I think my paperkey somewhere not sure though I did have it
but made new one after I got back into my account and now my app wasn't
working so uninstalled it again

**Comment by woodsjesse607** [comment]
Any advice would be greatly appreciated. I am on my device I originally used to create my keybase but when j install the app it treats it as if it were a new device. I have funds in my walllet I'm trying to get to without having to wait the 7 days but can't seem to figure out how. Is it even possible or do j have to just wait the 7 days?

**Comment by Brianetta** [comment]
You need a working device or a paper key.

In Keybase, the device isn't the physical phone hardware, it's the secret key data that the app stores on your phone. Clearing the data or uninstalling the app destroys the device; adding the same phone again is creating a new device.

I can see three devices in your sigchain. They are, or were, called `jesse`, `uhhh` and `woodsjesse`. If they're the same phone, then it is probably `woodsjesse`. If you have uninstalled since then, then you have no phone devices.

That leaves your paper key. It's a long list of English words starting with `bleak wealth`. If you no longer have the rest of it, then you are out of keys.

If you are out of keys, your account is lost. Without your paper key or one of your device keys, the only thing you can do is log in to the web site and request an account reset, which will revoke all of those four devices and add a new initial account creation to the end of your chain.
If you reset your account, you will need to re-prove your Twitter, Github and Reddit. You will need to re-claim your Bitcoin address. If you didn't export your Stellar wallet's secret key in advance, then your stellar wallet will be lost too.

To sum up, if you have really uninstalled Keybase from the only phone that had it, you *need* to find your paper key, otherwise you will have to reset your account. If you do not have the paper key, your account is already lost.

**Comment by woodsjesse607** [comment]
Ok thank you. I got back into my account. But my stellar wallet that u need
access to says I can't access it for 7 days it's on mobile only is there a
way around this or so I just have to wait the 7 days?

**Comment by Brianetta** [comment]
Are you waiting seven days for an account reset? If so, you will lose your wallet.

**Comment by woodsjesse607** [comment]
No. It just says for some reason that I can't send funds from a new device
until I wait 7 days.

**Comment by Brianetta** [comment]
Are you using the device called `woodjesse` to send the funds? If so, it's already five days old.

**Comment by woodsjesse607** [comment]
Yes well I was but my app messed up and had to reinstall it. So now it's
saying it's a new device. I haven't registered the new device yet. Is there
a way to create a new paper key or get into my account with just my
password? Sorry to bug ya with all the questions.

**Comment by woodsjesse607** [comment]
The other devices I was logged in on were on my girlfriend's phone and she
deleted the app also didn't realize I needed it.

**Comment by Brianetta** [comment]
No, there is no way to create a device or a paper key other than from an existing device or paper key. The only exception to this is creating your first device, with a new or a reset account.

**Comment by woodsjesse607** [comment]
Ok thanks

**Comment by woodsjesse607** [comment]
You guys I can log into my account on stellar.io now but and view secret Key and have key store file but still can't transfer or withdraw!?! Someone plz help me

**Comment by Brianetta** [comment]
If your problem is now on stellar.io you should probably contact them instead.

**Comment by soniaklick** [comment]
Well, i was in the same situation until i sorted professional help. These guys helped me recover my wallet private key [secrettruthhackers.imfast.io], what would i have done when my wallet had investors bitcoin in there? Thanks to the secrettruthhackers.
